packages/wrangler/src/cfetch/index.ts

```diff
@@ -31,18 +31,13 @@ export async function fetchResult<ResponseType>(
 	abortSignal?: AbortSignal,
 	apiToken?: ApiCredentials
 ): Promise<ResponseType> {
-	const json = await fetchInternal<FetchResult<ResponseType>>(
-		complianceConfig,
-		resource,
-		init,
-		queryParams,
-		abortSignal,
-		apiToken
-	);
+	const { response: json, status } = await fetchInternal<
```

Comment:
Side note: I really think the return type should be a cast... I mean it is but it should be explicit vs templated